backend/Application/Services/CategoryService.cs

```diff
@@ -22,4 +27,33 @@ public async Task<Response<List<GetCategoryResponse>>> GetAllAsync()
 
         return new Response<List<GetCategoryResponse>>(true, categories);
     }
+    public async Task<Response<CreateCategoryResponse>> CreateCategoryAsync(CreateCategoryRequest requestModel)
+    {
+        var categoryRepository = UnitOfWork.AsyncRepository<Category>();
+
+        var existPrefix = await categoryRepository.GetAsync(cat => cat.Prefix == requestModel.Prefix);
+
+        if (existPrefix != null)
+        {
+            return new Response<CreateCategoryResponse>(false, ErrorMessages.BadRequest);
```

Comment:
Return error message as the requirement when duplicate prefix. 
"Prefix is already existed. Please enter a different prefix"

Reply:
And also check for duplicate category name
"Category is already existed. Please enter a different category"